handbook/engineering/career-development/talent-review-process.md

```diff
@@ -52,10 +53,28 @@ Finally, we should have a [retrospective](../../retrospectives/index.md) after e
 
 Our [career development framework](framework.md) defines the levels that exist at Sourcegraph. Compensation for each of these levels is derived from Option Impact, a startup compensation tool that Sourcegraph and many other companies use to determine their compensation bands.
 
-## Considerations for promotion
+## Promotion requests
+
+Promotion requests are made immediately following the [talent review callibration meeting](#calibration-meeting).
+
+The Engineering Manager will complete a promotion request using the [promotion request template](https://docs.google.com/document/d/1DHyPtIooi0J2426iemzzTqLle9UBasrqLZPGET1O7Sc/edit) and submit it for approval.
+
+Following approval from the relevant decision-makers (EM, Director, VP), the promotion request is forwarded to the relevant teammates in PeopleOps (refer to [People Ops Communications Matrix](https://docs.google.com/spreadsheets/d/1JItBWbfKV9lr-LAmE19I0JMvu3Cvh0AdrEHDv-r1E2w/edit#gid=0)) for confirmation. Once confirmed by PeopleOps the promotion can be communicated to the teammate using the [promotion confirmation template](https://docs.google.com/document/d/1mkdJI6cSXWIHthSN_KQKBFSfxs9RQUg_4VCvwmk8FJQ/edit).
```

Comment:
What role does PeopleOps have here? Are they an _approver_ like the Director, or do they just need to be _informed_ to update payroll etc?

Reply:
@felixfbecker the latter - it's to ensure all necessary processes relating to the change get made in all the systems.